<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuedGameMenuWindow</class>
 <widget class="QWidget" name="ContinuedGameMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debate Lobby</string>
  </property>
  <property name="styleSheet">
   <string notr="true">     background-color:white;
</string>
  </property>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalVotesLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
</string>
   </property>
   <property name="text">
    <string>total votes:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnToMenuButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Game Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="highScoreLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
</string>
   </property>
   <property name="text">
    <string>high score:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dropoutButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Dropout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stage4Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Stage 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stage1Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Stage 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stage3Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Stage 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stage2Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Stage 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="introductionButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:blue;
	border: 1px solid #000000;
					 border-radius: 15px;
}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Introduction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="worldSettingsButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>620</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>World Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="FlagLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LibertyLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>261</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/head/Resources/Statue_Liberty.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myCharacterLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
